commands/account/use.ts

```diff
@@ -22,31 +22,31 @@ interface AccountUseArgs extends CommonArgs {
 export async function handler(
   args: ArgumentsCamelCase<AccountUseArgs>
 ): Promise<void> {
-  let newDefaultAccount = args.account;
+  let newDefaultAccountIdentifier = args.account;
 
-  if (!newDefaultAccount) {
-    newDefaultAccount = await selectAccountFromConfig();
-  } else if (!getAccountId(newDefaultAccount)) {
+  if (!newDefaultAccountIdentifier) {
+    newDefaultAccountIdentifier = await selectAccountFromConfig();
+  } else if (!getConfigAccountIfExists(newDefaultAccountIdentifier)) {
     logger.error(
       i18n(`${i18nKey}.errors.accountNotFound`, {
-        specifiedAccount: newDefaultAccount,
-        configPath: getConfigPath()!,
+        specifiedAccount: newDefaultAccountIdentifier,
+        configPath: getConfigFilePath(),
       })
     );
-    newDefaultAccount = await selectAccountFromConfig();
+    newDefaultAccountIdentifier = await selectAccountFromConfig();
   }
 
-  trackCommandUsage(
-    'accounts-use',
-    undefined,
-    getAccountId(newDefaultAccount)!
-  );
+  const newDefaultAccount = getConfigAccountIfExists(
```

Comment:
Technically `getConfigAccountIfExists` can throw now. Should we wrap this in a try/catch too? Maybe this is a +1 for allowing a way to call this without it throwing? I suggested adding an argument to these to allow for a "no throw" variation, but another option could be to export versions of these utils that catch errors and return null instead of throwing. Like `getConfigAccountOrNull`

Reply:
`getConfigAccountIfExists` is a new function I added specifically for occasions like this where you don't want to throw if the account isn't there. Technically it can still throw if there's no config at all, but I think that's the probably the behavior we want.